content/working_with/console/widgets/filters.md

```diff
@@ -40,9 +40,28 @@ Add, edit and clone operation modals share a common component for defining filte
 The component presents a list of rows, each representing a single filter rule. Each row contains three inputs:
 
 * Rule type selection dropdown - selecting the context of the rule which can be based on labels or supported deployment attributes such as blueprint ID, site name, or creator.
-* Rule operator dropdown. The set of available operators to choose from depends on the selected rule type.
+* Rule operator dropdown. The set of available operators to choose from depends on the selected rule type. See Table 1. and Table 2. below for details.
 * Value input (for attribute rules) or key/value input(s) (for label rules). 
 
+| Operator label in the UI       | Operator name in the API/CLI        |
```

Comment:
Just wondering.  We are using 'label' word as a term for cfy label entity, for example in L44 ('label rules'). But here we are using it as a term for string occurring in the UI. Isn't it confusing? Wouldn't it be better to name this table column 'Operator name in the UI'?

Reply:
+1 
I had the same comment :)

Reply:
Make sense. I have refactored it even more 🙂 Hope it's more user-friendly now.